pkg/controllers/provisioningv2/rke2/provisioningcluster/template.go

```diff
@@ -491,6 +491,7 @@ func rkeControlPlane(cluster *rancherv1.Cluster) (*rkev1.RKEControlPlane, error)
 			ETCDSnapshotRestore:      cluster.Spec.RKEConfig.ETCDSnapshotRestore.DeepCopy(),
 			ETCDSnapshotCreate:       cluster.Spec.RKEConfig.ETCDSnapshotCreate.DeepCopy(),
 			RotateCertificates:       cluster.Spec.RKEConfig.RotateCertificates.DeepCopy(),
+			RotateEncryptionKeys:     cluster.Spec.RKEConfig.RotateEncryptionKeys.DeepCopy(),
```

Comment:
I know this isn't really part of your PR, but can we put an `rkeConfig := cluster.Spec.RKEconfig.DeepCopy()` above and use the pieces here without all the `DeepCopy` calls?